change/@fluentui-react-5a84fff9-977e-458d-94a0-c8e5bd1c8930.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The changes appear to be because some icons didn't load in the baseline screenshot (rendering unicode boxes), and they loaded in the candidate screenshots. The changes are unrelated to this change, and the candidate ones look like the correct ones.